openfe/setup/atom_mapping/perses_mapper.py

```diff
@@ -25,6 +25,30 @@ class PersesAtomMapper(LigandAtomMapper):
     allow_ring_breaking: bool
     preserve_chirality: bool
     use_positions: bool
+    coordinate_tolerance: float
+
+    def _to_dict(self) -> dict:
+        # strip units but record values
+        return {
+            "allow_ring_breaking": self.allow_ring_breaking,
+            "preserve_chirality": self.preserve_chirality,
+            "use_positions": self.use_positions,
+            "coordinate_tolerance": self.coordinate_tolerance.value_in_unit(
```

Comment:
This would definitely be a lot cleaner if we switched to OpenFF units instead and converted to openmm units as necessary. Probably best in a follow up PR though.

Reply:
Yes +1 for a consistent unit package in the code base!